docs/docusaurus.config.js

```diff
@@ -25,6 +25,7 @@ const versionLabels = {
 
 module.exports = {
   customFields: {
+    rootEditUrl: 'https://github.com/rasahq/rasa/', // NOTE: this could be directed to a specfic README or a heading within a README
```

Comment:
should we add . FIXME that references this issue: https://github.com/RasaHQ/rasa/issues/7011 ?

also I think the NOTE is not that useful 🤷

Reply:
OK, have updated this to a FIXME with issue ref and more explanation about future solution options